### PR TITLE
CC3220SF SimpleLink SDK 1.6 Support

### PR DESCRIPTION
**Pull request by DellaBitta**
[ Description ]
Tested builds hosted on Windows with new CC3220SF Simplelink SDK version 1.6 and XDC Tools version 3_50_04_43_core.  Replaced Provisioning Task with Network Interface pulled from other examples, which simplifies the example application.  Implementation now handles the lost of WiFi and subsequent re-connection, where as the previous example did not.

[ JIRA ]
None.

[ Reviewer ]
@atigyi 

[ QA ]
Built on Windows.
Tested Debug Version loaded onto device via Code Composer Studio.
Tested Flashed Version burnt into the firmware via Uniflash. 
Went through the Tutorial to clarify some problems.

[ Release Notes ]
- CC3220SF tutorial is now built and tested with the Simplelink SDK v1.6, XDC Tools 3_50_04_43_core, and TI CC3220SF Service Pack sp_3.6.0.3_2.0.0.0_2.2.0.6.bin.
- CC3220SF Tutorial Application now detects WiFi signal loss and attempts to reconnect.

### CODE REVIEW

File: examples/cc3220sf/xively_demo/.gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-./Release
-./Demo
\ No newline at end of file
+/Release
+/Debug
```
Comment (atigyi) [inline examples/cc3220sf/xively_demo/.gitignore]: 👍

File: examples/cc3220sf/xively_demo/.project [hunk 2]
```diff
@@ -29,7 +29,7 @@
 	<variableList>
 		<variable>
 			<name>XIVELY_LIBRARY_C_ROOT</name>
-			<value>file:/Users/atigyi/git/xi/xc/xively-client-c-rhunter</value>
+			<value>file:/E:/dev/xively/xively-client-c</value>
```
Comment (atigyi) [inline examples/cc3220sf/xively_demo/.project]: 👊 + 😄

Comment (DellaBitta) [reply examples/cc3220sf/xively_demo/.project]: hehe

File: examples/cc3220sf/wolfssl_ccs/.cproject [hunk 3]
```diff
@@ -15,21 +15,21 @@
 			<storageModule moduleId="cdtBuildSystem" version="4.0.0">
 				<configuration artifactExtension="lib" artifactName="${ProjName}" buildProperties="" cleanCommand="${CG_CLEAN_CMD}" description="" id="com.ti.ccstudio.buildDefinitions.TMS470.Debug.459548823" name="Debug" parent="com.ti.ccstudio.buildDefinitions.TMS470.Debug">
 					<folderInfo id="com.ti.ccstudio.buildDefinitions.TMS470.Debug.459548823." name="/" resourcePath="">
-						<toolChain id="com.ti.ccstudio.buildDefinitions.TMS470_17.6.libraryDebugToolchain.995820573" name="TI Build Tools" superClass="com.ti.ccstudio.buildDefinitions.TMS470_17.6.libraryDebugToolchain" targetTool="com.ti.ccstudio.buildDefinitions.TMS470_17.6.library.librarianDebug.1794465251">
-							<option id="com.ti.ccstudio.buildDefinitions.core.OPT_TAGS.1003748928" superClass="com.ti.ccstudio.buildDefinitions.core.OPT_TAGS" valueType="stringList">
+						<toolChain id="com.ti.ccstudio.buildDefinitions.TMS470_16.9.libraryDebugToolchain.1294023494" name="TI Build Tools" superClass="com.ti.ccstudio.buildDefinitions.TMS470_16.9.libraryDebugToolchain" targetTool="com.ti.ccstudio.buildDefinitions.TMS470_16.9.library.librarianDebug.115484381">
```
Comment (atigyi) [inline examples/cc3220sf/wolfssl_ccs/.cproject]: 👍

**Comment by atigyi** [comment]
Approved 👍

**Pull request merged**